### PR TITLE
fix: mobile render

### PR DESCRIPTION
**Pull request by vcellu**
This Pr updates the readme to include instructions on how to build and run on iOS simulator.
It also fixes an issue in mobile where the picasso chart needs to be initialized.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -60,8 +60,19 @@ const supernova = (env) => {
...
+            env,
+            picassoQ,
+            selectionsApi: selections,
+            showLegend,
+            invalidMessage,
+            translator,
+            options,
+          });
+          const data = { type: 'q', data: layout.qHyperCube, key: 'qHyperCube' };
+          const c = pic.chart({ element, settings, data });
```
Comment (T-Wizard) [inline src/index.js:75]: why does it need to render with data here?
It make it harder to add fetching extra async data harder (needed to fix more advanced coloring)

File: README.md [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
 Treemap Supernova
 
+## Mobile
+### Preamble
+Mobile uses react-native.   As such all javascript should work on both the web and mobile. However react-native uses a custom JavaScript engine called Hermes.  For rendering picasso charts, mobile uses a hightly optimized rendering engine, called [Helium](https://github.com/qlik-oss/react-native-helium), that uses a GPU backend for Skia.  Skia is the same 2D graphics library used for Canvas in Chrome and Android.  So again, things must be tested to ensure compatiblity.
+
+### Testing mobile
+#### Pre-reqs
+1. Install [react-native dev environment](https://reactnative.dev/docs/0.68/environment-setup). This includes Xcode and Android Studio.
+2. A QCS Account with an API Key
```
Comment (T-Wizard) [inline README.md:11]: I am a bit unsure what this means (any link with more information)
(I have not tried to look for more info myself yet, maybe it is easy to find if you search for it)

File: README.md [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+3. An app with some treemaps
+
+
+#### Running
+** Please Note this can take a while to build if it's the first time **
+1. `yarn prepare-mobile`
+2. `yarn ios`
```
Comment (T-Wizard) [inline README.md:18]: is this relevant on a windows machine? (where I can only install Android Studio and not xcode)
or should I run something else instead.

File: src/index.js (hunk 1 above)
Comment (vcellu) [reply src/index.js:75]: Can you point me where this is causing an issue to do async fetches? I would like to understand the problem before I address this further.

File: README.md (hunk 2 above)
Comment (vcellu) [reply README.md:11]: The link to setup the react native dev env is provided in step 1.
How to setup an API Key is here https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Admin/mc-generate-api-keys.htm

File: README.md (hunk 3 above)
Comment (vcellu) [reply README.md:18]: We are aware that not everyone has a complete mobile dev environment, and we are exploring options to automate this in the cloud.  I will add the "Xcode" requirement explicitly in the pre reqs

File: src/index.js (hunk 1 above)
Comment (vcellu) [reply src/index.js]: Done.

File: src/picasso-def/components/treemap.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ import { treemap as d3Treemap } from 'd3-hierarchy';
 import { color as d3Color } from 'd3-color';
 import { createTextLabels, displayInvalidMessage, createOverlayLabel } from './labels';
 import { TREEMAP_DEFINES } from './defines';
+import { setupBrushes } from './setupBrushes.native';
```
Comment (T-Wizard) [inline src/picasso-def/components/treemap.js:5]: should it import the .natve file?

**Review by T-Wizard — APPROVED** [review]

**Pull request merged**